Calculator/Calculator/ViewController.swift

```diff
@@ -7,12 +7,208 @@
+        
+        let stackLabel = UILabel()
+        stackLabel.text = operatorStackLabel + " " + operandStackLabel
+        stackLabel.numberOfLines = 0
+        stackLabel.adjustsFontForContentSizeCategory = true
+        stackLabel.font = UIFont.preferredFont(forTextStyle: .title3)
+        stackLabel.textColor = .white
+        stackView.addArrangedSubview(stackLabel)
+        setScrollView()
+        
+    }
+    
+    private func setScrollView() {
+        let bottomOffset: CGPoint = CGPointMake(0, scrollView.contentSize.height)
+        scrollView.setContentOffset(bottomOffset, animated: false)
+    }
+    
+    private func applyDecimalPoint(number: String) -> String {
+        numberFormatter.numberStyle = .decimal
+        numberFormatter.maximumSignificantDigits = 20
+        guard let operand = Double(number) else { return "" }
+        guard let result = numberFormatter.string(from: NSNumber(value: operand)) else { return "" }
+        
+        return result
+    }
+    
+    private func checkSign() {
+        
+        guard var currentNumber = numberInput.text,
+              currentNumber != "0" else { return }
+        
+        if let minus = currentNumber.firstIndex(of: "-") {
+            currentNumber.remove(at: minus)
+            numberInput.text = currentNumber
+        } else {
+            numberInput.text = "-" + currentNumber
+        }
+    }
+    
+    private func resetStackView() {
+        stackView.arrangedSubviews.forEach{ $0.removeFromSuperview() }
+    }
+    
+    private func resetNumberInput() {
+        currentNumber = "0"
+    }
+    
+    private func resetOperatorInput() {
+        operatorInput.text = " "
+    }
+    
+    private func handleDivideError() {
+        
+        do {
+            let resultLabel = try checkDecimalPoint()
+            numberInput.text = resultLabel
+            isFinishedCalculating = true
+        } catch CalculatorError.divideByZero {
+            numberInput.text = "NaN"
+        } catch {
+            print("계산오류")
+        }
+    }
+    
+    private func checkDecimalPoint() throws -> String {
+        
+        let resultLabel: String = calculate()
+        let dividedValue = resultLabel.components(separatedBy: ".")
+        guard !dividedValue[0].isEmpty || dividedValue[1] == "0" else {
+            throw CalculatorError.calcuate
+        }
+        numberFormatter.minimumFractionDigits = 0
+        guard let number = Double(resultLabel) else { return "" }
+        guard let formatterNumber = numberFormatter.string(from: NSNumber(value: number)) else { return "" }
+        return formatterNumber
+    }
+    
+    private func calculate() -> String {
+        
+        let calculateItem = arrangeCalculateItems()
+        var formula = ExpressionParser.parse(from: calculateItem)
+        let result = formula.result()
+        guard let resultValue = result else { return "" }
+        let resultLabel = String(resultValue)
+        return resultLabel
```

Comment:
이렇게 간략하게도 작성할 수 있습니다!

```suggestion
        guard let resultValue = formula.result() else { return "" }

        return String(resultValue)
```

Reply:
코드가 너무 길어서 줄이고 싶어서 고민해봐도 생각이 안났는데...😅 감사합니다 주디!